board_client/animations/utils.py

```
@@ -83,6 +90,180 @@ def falloff(dy, width):
+    """Perform garbage collection at strategic times"""
+    gc.collect()
+    
+def get_memory_info():
+    """Get current memory usage info"""
+    try:
+        import gc
+        return {
+            'free': gc.mem_free(),
+            'allocated': gc.mem_alloc()
+        }
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)